source/hardware/controllers/hercules_djcontrol_MIX.rst

```diff
@@ -0,0 +1,117 @@
+Hercules DJControl MIX
+==============================
+
+-  `Manufacturer’s product page <https://www.hercules.com/en-us/product/djcontrol-mix/>`__
+-  `Manufacturer’s support and downloads page <https://support.hercules.com/en/product/djcontrolmix-en/>`__
+-  `Forum thread <https://mixxx.discourse.group/t/hercules-contrl-mix-mapping/26581>`__
+
+.. versionadded:: 2.4
+
+Compatibility
+-------------
+
+This controller is a class compliant USB MIDI, so it can be used without any special drivers on GNU/Linux, Mac OS X, and
+Windows.
+
+.. note::
+   In order to use this mapping, the DJControl MIX must be physically connected to the computer through USB. It was not conceived to use the controller connected via Bluetooth.
+
+Sound Card Setup
+----------------
+
+This device does not have a built in audio interface.
+
+.. seealso::
+   Please refer to :ref:`example setups section <setup-laptop-and-external-card>` for more details about the audio configuration in Mixxx.
+
+Mapping description
+-------------------
+
+Decks
+~~~~~
+
+===============================================================  ==========================================
+Control                                                          Function
+===============================================================  ==========================================
+:hwlabel:`SYNC` button                                           Toggle :ref:`Sync Lock <master-sync>`.
```

Comment:
```suggestion
:hwlabel:`SYNC` button                                           Toggle :ref:`Sync Lock <sync-lock>`.
```
`master-sync` was renamed to `sync-lock`: https://manual.mixxx.org/2.4/en/chapters/djing_with_mixxx#sync-lock
Please adjust your link, to fix the build-check warning!